tests/conftest.py

```diff
@@ -99,6 +99,7 @@ def env_type(request: pytest.FixtureRequest) -> str:
 @pytest_asyncio.fixture(scope="session")
 async def env(env_type: str) -> AsyncGenerator[WorkflowEnvironment, None]:
     if env_type == "local":
+        http_port = 7243
```

Comment:
Why not use the official way of starting/cancelling Nexus operations via workflows instead of this less-official HTTP manner? Is there something that HTTP gives you that Nexus client in the workflow doesn't that you need to test?